Sample/SwiftUI/CustomEventView.swift

```diff
@@ -0,0 +1,132 @@
+import SwiftUI
+import RInAppMessaging
+
+@available(iOS 13.0, *)
+struct CustomEventView: View {
+
+    @State private var isAlertPresented = false
+    @State private var isErrorAlertPresented = false
+    @State private var eventName: String = ""
+    /// Init with one empty attribute for UI purpose
```

Comment:
Please replace this comment by:
```
/// - Note: Init with one empty attribute for UI purpose
```

Reply:
Updated here [b3a87fc](https://github.com/rakutentech/ios-inappmessaging/pull/278/commits/b3a87fc8fee72033dd8340384020d67b44568791).Updated here [b3a87fc](https://github.com/rakutentech/ios-inappmessaging/pull/278/commits/b3a87fc8fee72033dd8340384020d67b44568791).

Reply:
Updated here [b3a87fc](https://github.com/rakutentech/ios-inappmessaging/pull/278/commits/b3a87fc8fee72033dd8340384020d67b44568791).